### PR TITLE
Add software procurement process code to projects page

### PR DESCRIPTION
**Pull request by ElenaFdR**
This is a starting point. Obviously, this would be tweaked as soon as there's more than 1 process code, or more process code related content (like a white paper).

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -38,6 +38,14 @@ It's a self study course and offers a certification of completion to those who f
 To increase the accessibility of the Standard for Public Code, the [translations repository](https://github.com/publiccodenet/community-translations-standard) allows for the community to collaborate on translations of it.
 It is possible to add more translations as more communities find it useful and are willing to translate and review.
 
+### [Software procurement process code](https://processcode.publiccode.net/)
+
+Process codes are flexible, collaborative, and adaptive guidebooks for a new government process.
+
+The software procurement process code helps public sector employees source the best possible software. It walks through government contracting step-by-step, aligning legitimate public procurement with the cutting-edge best practices of development.
```
Comment (ericherman) [inline index.md:45]: For `git diff` friendliness, we typically have one-line per sentence, and allow long lines.

Comment (ElenaFdR) [reply index.md]: Done!

**Comment by ElenaFdR** [comment]
@mcarlson @matthewclaudel Any feedback? Can I just go ahead and merge this?

**Comment by ElenaFdR** [comment]
Apologies, I'd forgotten that processcode.publiccode.net currently 404s. So please take a look and give feedback/approve, but we'll wait with publishing til we have a live destination link.

**Comment by ericherman** [comment]
It seems that the process code repository was changed back to be only https://publiccodenet.github.io/processcode.publiccode.net/ , no longer https://processcode.publiccode.net/ so we should change these links to reflect that or we need to change the GitHub repository settings.

**Comment by nebster** [comment]
Max said he set it up for processcode.publiccode.net ?  That should be the URL.

Ben Cerveny, President
Foundation for Public Code <https://publiccode.net/>
***@***.*** | @neb

**Review by ericherman — APPROVED** [review]

**Pull request merged**